docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
         build:
             context: .
             dockerfile: Dockerfile
+            args:
+                commit: ${IMAGE_TAG}
```

Comment:
@giorgiosironi This seems to be the fix (not setting means it always gets the `latest` assets).

Reply:
Extracted into #849.